<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-B4SFM7J" failures="1" tests="2" name="Test" time="3.652" errors="0" timestamp="2025-03-08T20:20:55 IST">
  <testcase classname="Listener_handling.TestClass" name="testMethod1" time="0.01"/>
  <testcase classname="Listener_handling.TestClass" name="testMethod2" time="0.967">
    <failure type="java.lang.AssertionError" message="Intentional failure in testMethod2 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Intentional failure in testMethod2 expected [true] but found [false]
at Listener_handling.TestClass.testMethod2(TestClass.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
</testsuite> <!-- Test -->
